prow/config/branch_protection.go

```diff
@@ -444,11 +444,10 @@ func (c *Config) unprotectedBranches(presubmits map[string][]Presubmit) []string
 //     because any branches not explicitly specified in the configuration will be unprotected.
 func (c *Config) BranchProtectionWarnings(logger *logrus.Entry, presubmits map[string][]Presubmit) {
 	if warnings := c.reposWithDisabledPolicy(); len(warnings) > 0 {
-
-		logger.WithField("repos", strings.Join(warnings, ",")).Warn("The following repos define a policy, but have protect: false")
```

Comment:
Are you sure? AFAIK those are essentially misconfigurations
/cc @petr-muller

Reply:
yeah I believe so, see https://github.com/kubernetes/test-infra/issues/21983#issuecomment-827969398

Reply:
I think the PR is correct, I think "policy" here is usually inferred from the presence of required jobs, and the fact that you have required jobs does not always mean you want to manage BP.